### PR TITLE
Fix:overlapping of the save as draft (button)

### LINKED ISSUE #5218: The Save as draft button overlapping in mobile view
**Describe the bug**
The save button is overlapping on the mobile view see in the screenshot for the better understanding

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'https://eventyay.com/create'
2. Click on 'inspect elment'
3. Scroll down to 'see save as draft button'

**Screenshots**

**Smartphone (please complete the following information):**

### PR DESCRIPTION
**Pull request by Raunaksingh100**
Fixes #5218 

#### Short description of what this resolves:
fixed the overlapping of the "save as Draft button"  in mobile view and it does not affect the  desktop site
see in the screenhot for desktop and mobile site for both 

### screenshot before Desktop and mobile view -
1. 
1 b

2. 

### screenshot after desktop and mobile view--
1. 
 1 b
2.  
1fix

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Raunaksingh100** [comment]
@mariobehling and @iamareebjamal plz review

File: app/styles/pages/public-event.scss [hunk 1]
```diff
@@ -185,3 +185,7 @@
 .event-map > iframe.g-map {
   height: 300px;
 }
+
+button.ui.blue.button.right.labeled.icon.small.right.aligned {
+  margin-top: 5px;
+}
```
Comment (snitin315) [inline app/styles/pages/public-event.scss:191]: Can you fix it using semantic UI classes or predefined classes?

Comment (Raunaksingh100) [reply app/styles/pages/public-event.scss:191]: @snitin315 but this bug is on mobile devices

Comment (Raunaksingh100) [reply app/styles/pages/public-event.scss:191]: @snitin315 the bug in mobile view not the desktop site

Comment (snitin315) [reply app/styles/pages/public-event.scss:191]: `device.isMobile`

Comment (maze-runnar) [reply app/styles/pages/public-event.scss:191]: instead of margin-top use mt-8 class

File: app/components/forms/wizard/wizard-footer.hbs [hunk 2]
```diff
@@ -22,6 +22,7 @@
       </button>
     {{/if}}
     {{#if (eq @event.state 'draft')}}
+      <div class="mt-8 {{if device.isMobile }}"></div>
```
Comment (maze-runnar) [inline app/components/forms/wizard/wizard-footer.hbs:25]: {{if device.isMobile }} why this is here when it's for nothing

Comment (maze-runnar) [reply app/components/forms/wizard/wizard-footer.hbs:25]: And build is failing too due to this

Comment (Raunaksingh100) [reply app/components/forms/wizard/wizard-footer.hbs:25]: > And build is failing too due to this

@maze-runnar so the button is overlapping in mobile view , that's why i add can you tell what i do

Comment (maze-runnar) [reply app/components/forms/wizard/wizard-footer.hbs:25]: {{if device.isMobile 'mt-8'}}

File: app/components/forms/wizard/wizard-footer.hbs [hunk 3]
```diff
@@ -23,12 +23,14 @@
     {{/if}}
     {{#if (eq @event.state 'draft')}}
       <button class="ui blue button right labeled icon small right aligned {{if @loading 'disabled'}}" type="button" {{action @onSaveDraft}}>
-        {{t 'Save as draft'}} <i class="save icon"></i>
+      {{#if this.device.isMobile 'mt-8'}}
+      {{t 'Save as draft'}} <i class="save icon"></i>
```
Comment (maze-runnar) [inline app/components/forms/wizard/wizard-footer.hbs:27]: what are you trying to do, what is your logic? I don't think u even know what this code is doing.

**Pull request closed without merging**